vignettes/gDRcore.Rmd

```diff
@@ -39,14 +39,21 @@ a series or the same number of elements, the attributes (columns of the table) s
 ## Drug processing 
 For drug response data, the input files need to be merged such that each measurement (data) is associated with the right metadata (cell line properties and treatment definition). Metadata can be added with the function `cleanup_metadata` if the right reference databases are in place.
 
-When the data and metadata are merged into a long table, the wrapper function `runDrugResponseProcessingPipeline` can be used to generate an MAE with processed and analyzed data. In practice runDrugResponseProcessingPipeline does the following steps:
```

Comment:
Maybe also this chunk wit loading libraries could be with `echo=FALSE`?

Reply:
done.